authority/provisioner/scep.go

```diff
@@ -0,0 +1,104 @@
+package provisioner
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// SCEP is the SCEP provisioner type, an entity that can authorize the
+// SCEP provisioning flow
+type SCEP struct {
+	*base
+	Type string `json:"type"`
+	Name string `json:"name"`
+
+	ForceCN           bool     `json:"forceCN,omitempty"`
+	ChallengePassword string   `json:"challenge,omitempty"`
```

Comment:
I hadn't used that function of the Step CLI yet, but you're most probably right. When I tried this however, I didn't get the output that I would expect for the SCEP provisioner; only the default JWK one was shown so far, but I guess that's due to my local Step CLI version not understanding SCEP yet? Although the code in that repo seems to show it should show the JSON, so I'm not fully sure why I didn't get the SCEP as JSON output.

The challenge should indeed be considered secret. As it is used now, it's a shared secret amongst all SCEP clients connecting to this CA. This is in line with the RFC. Of course it would be nice to have multiple secrets, perhaps even unique per client, but that would require additional storage or a different way to provide these to Step.

Anyway, I made a relatively small change that redacts the output in the provisioner `Init` function. This seemed to be the solution that required the least amount of changes and seems effective so far.